README.md

```diff
@@ -171,16 +190,18 @@ order by internal_id desc;
 
 ## Roadmap
 
-1. Switch to application level chat protocol. This will allow to separate physical server connection management from logical chat contacts, and to support all common chat functions.
-2. SMP queue redundancy and rotation in SMP agent protocol.
-3. Symmetric groups support in SMP agent protocol, as a foundation for chat groups.
-4. Delivery confirmation in SMP agent protocol.
-5. Multi-agent/device data synchronisation - to use chat on multiple devices.
+The consumer ready system will have these parts implemented:
+
+1. Application level chat protocol. This will allow to separate physical server connection management from logical chat contacts, and to support all common chat functions. Currently in progress in [v4 branch](https://github.com/simplex-chat/simplex-chat/tree/v4)
+2. Symmetric groups support in SMP agent protocol, as a foundation for chat groups.
+3. SMP queue redundancy and rotation in SMP agent protocol.
+4. Message delivery confirmation in SMP agent protocol.
+5. Multi-agent/device data synchronization - to use chat on multiple devices.
 6. Synchronous streams support in SMP and SMP agent protocols, to support file transfer.
-7. Terminal chat UI and mobile apps.
-8. Scripts for simple SMP server deployment to hosting providers: Linode, Digital Ocean and Heroku.
+7. Desktop and mobile apps.
+8. Scripts for simple SMP server deployment to hosting providers: Linode ([done](https://github.com/simplex-chat/simplexmq#deploy-smp-server-on-linode)), Digital Ocean and Heroku.
 9. Public broadcast channels.
-10. Optional public contact/group addresses using DNS to establish connections, but not using it to send and receive messages - in this way you will keep all your contacts and groups even if you lose the control of the domain.
+10. Optional public contact/group addresses using DNS-based contact addresses (like email) to establish connections, but not using it to route messages - in this way you will keep all your contacts and groups even if you lose the control of the domain.
 
 ## Disclaimer
```

Comment:
we have two disclaimers now - move this to the upper one? also this one's removed from table of contents

Reply:
I actually thought to completely drop the second and apparently forgot... It feels like the first one is stronger :)